frontend/src/modules/auth/auth-routes.js

```diff
@@ -26,19 +26,19 @@ export default [
         name: 'signup',
```

Comment:
I would also add one extra title to this file, to the Profile Settings one which is at the bottom of `auth-routes.js` file.
I think the title can be `Profile Settings`. Do you mind making that change?

Reply:
Added.